filament/backend/src/opengl/CocoaTouchExternalImage.mm

```diff
@@ -228,6 +229,21 @@ void main() {
     return texture;
 }
 
+CVOpenGLESTextureRef CocoaTouchExternalImage::createTextureFromImage(CVPixelBufferRef image, GLuint
```

Comment:
This doesn't need to be a new method. Add the `GLuint pixelFormat` argument to the original `createTextureFromImage` and pass `GL_RGBA` for the other calls.